### PR TITLE
Request for Code Review: Building with Backgrounds & Gradients

### PR DESCRIPTION
**Pull request by edieatha**
## Apple Page Clone:
1. Used background image for header image.
2. Applied a gradient on navigation buttons.
3. Used positioning 'absolute' on all sections of the page.
4. Aligned footer and aside elements with flex.
5. Added images and text styles.

### CODE REVIEW

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @edieatha  👋

You've done quite well on this.
The project looks quite good to go, but for a few changes.

Kindly make the following changes.

- [ ] Required - As this project is focused on backgrounds and gradients, please make **CONVERT** image elements in the project to backgrounds. 

- [x] Required - Also the search input is expected to be a form input field. Kindly change this, and assign the search icon as a background too.
ice_screenshot_20200414-001446

- [x] Required - Kindly change the images you have in the screenshot below👇 into text elements. You should make them text elements in the project.
ice_screenshot_20200414-001855

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+						alt="What will your verse be?" width="980" height="90"></a></div>
+				<div class="summary"><a href="#"><img class="small"
+						src="images/subhead_small.png" alt="Everyone has something to share."
+			                          width="980" height="82"></a></div>
+				<div class="watch">
+					<a href="#"><img class="small"
+						src="images/cta_your_verse_01.png" alt="Your verse"
+						 width="980" height="34"></a> </div>
+				</div>
+
```
Comment (mikenath223) [inline index.html]: Convert these image elements to text elements instead.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+					   alt="iPhone 5c"></a>
+		</div>
+		<div>
+			<a href="#"><img src="images/promo_30_years.jpg"
+					 alt="Happy Birthday Apple"></a>
+		</div>
+		<div>
+			<a href="#"><img src="images/supplier_responsibility.jpg"
+					 alt="Supplier Responsibility"></a>
+		</div>
```
Comment (mikenath223) [inline index.html:73]: Convert the images here also to text elements

File: index.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            <li class="gn-ipad"><a href="#"><span>iPad</span></a></li>
+            <li class="gn-itunes"><a href="#"><span>iTunes</span></a></li>
+            <li class="gn-support"><a href="#"><span>Support</span></a></li>
+          </ul>
+          <div class="globalsearch">
+            <form class="g-search">
+                <div class="magnify">
+                    <img src="images/search-icon.png" alt="search" height="13" width="13">
+                </div>
+            </form>
```
Comment (mikenath223) [inline index.html]: Kindly change this into a search input field. Also fix the HTML validation errors in the HTML code with this [tool](https://validator.w3.org/nu/#file).

**Comment by edieatha** [comment]
## TSE Code Review Changes

1. Converted image elements in HTML to background CSS elements.
2. Changed search element to input type element
3. Changed text images to actual text elements
4. Validated HTML

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  @edieatha  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**

Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**